scripts/create_patches_files.py

```diff
@@ -96,11 +115,16 @@ def parse_patches(patches: List[Dict[str, Any]], patch_id: Union[None, str] = No
             prev_patchworks_link.append(f"{patch['name']}\t{patch['web_url']}\t{patch['id']}\n")
             all_patchworks_links[found_series].append(prev_patchworks_link)
 
-        if patch_id is None and ("risc-v" in patch["name"].lower() or "riscv" in patch["name"].lower()):
-            print(f"Patch {patch['name']} is a risc-v patch")
+        if patch_id is None and interesting_patch(patch):
+            print(f"Patch {patch['name']} is an interesting patch")
             riscv_download_links[found_series].append(all_download_links[found_series][-1])
             riscv_patchworks_links[found_series].append(all_patchworks_links[found_series][-1])
 
+        # Old check, used for asserts
+        if patch_id is None and ("risc-v" in patch["name"].lower() or "riscv" in patch["name"].lower()):
+            riscv_title_patch_links.append(tuple(all_download_links[found_series][-1]))
```

Comment:
why do we cast `all_download_links[found_series][-1]` as a tuple if we end up casting `links` to a tuple later during the assert?

Reply:
This is the left side of that assert - The right side still needs casting so we do that in the assert.
I can convert it all to use tuples if that's preferred.

Reply:
right I think I'm just confused as to why the left and right sides needs to be casted to tuples. Since they are both lists before the cast, can't we compare them that way or does the `assert any` not work with list vs list comparisons?

Reply:
Ah I converted it since tuples were hashable but that ended up not being relevant for this solution. Good catch - I'll switch it back to just lists.